php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -259,4 +259,18 @@ public static boolean isNullOrEmptyString(ExpressionTree tree) {
     return false;
   }
 
+  public static Tree getParentOfKind(Tree tree, Kind kind) {
```

Comment:
A usual practice at SonarSource is to add a `@CheckForNull` annotation on methods which can return null.